src/content/1.7/development/components/grid/columns-reference/employee-name-with-avatar.md

```diff
@@ -25,13 +30,12 @@ Employee's name field.
 use PrestaShop\PrestaShop\Core\Grid\Column\Type\Employee\EmployeeNameWithAvatarColumn;
 use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollection;
 
-$column = new EmployeeNameWithAvatarColumn('severity');
-$column->setName('Severity (1-4)');
-$column->setOptions([
-     'field' => 'severity',
-     'with_message' => true,    // enable severity messages
+$employeeColumn = new EmployeeNameWithAvatarColumn('employee');
```

Comment:
Did not see this class `EmployeeNameWithAvatarColumn` pass in a PR :/
Avatar should have been an option :/

Reply:
+1, who have validated this new column type? this break MVC principles /C @sarjon @matks

Reply:
you're probably right, but then the code should be fixed not the doc ^^